openhands_resolver/resolve_issues.py

```diff
@@ -197,9 +198,16 @@ def get_instruction(
     issue: GithubIssue,
     prompt_template: str,
     repo_instruction: str | None = None,
+    issue_type: str | None = None
```

Comment:
```suggestion
    issue_type: str,
    repo_instruction: str | None = None,
```
We probably don't need `None` issue types, see below.